src/include/units/quantity.h

```diff
@@ -204,22 +204,22 @@ class quantity {
   }
 
   constexpr quantity& operator*=(const rep& rhs)
-    requires requires(rep a, rep b) { { a *= b } -> std::same_as<rep&>; }
+    requires requires(rep a, const rep b) { { a *= b } -> std::same_as<rep&>; }
```

Comment:
Shouldn't it be `const rep&`?

Reply:
The `b` in the expression is already an lvalue of type `const rep`. Adding `&` to the types can help with clarity, though. Like
```suggestion
    requires requires(rep& a, const rep& b) { { a *= b } -> std::same_as<rep&>; }
```

Reply:
I just wondered if `const rep b` does not imply that `b` has to be copy- or move-constructible, but probably not...

Reply:
The optional parameter list indeed implies nothing about parameter passing.